appveyor.yml

```diff
@@ -22,6 +22,7 @@ environment:
 install:
   - ps: >-
       $Env:PATH += ';C:\msys64\usr\bin'
+  - if "%TARGET%" == "x86_64-pc-windows-msvc" call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" x86_amd64
```

Comment:
You'll probably need to copy the rest of the setup for this from gutenberg
https://github.com/Keats/gutenberg/blob/master/appveyor.yml

Reply:
yeah, I'm trying to figure out what's wrong ^^ bear with me, it will be a git mess in the end x)